### PR TITLE
Implement actions for role including tests

### PR DESCRIPTION
**Pull request by r-peschke**
Implements the actions for role-object

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
As discussed: Plase make `permissions` unique. E.g. if the user sends `["a", "a", "b"]` the result should be `["a", "b"]` (the ordering is irrelevant, so `["b", "a"]` is also fine).

File: openslides_backend/action/role/deduplicate_permissions_mixin.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from ..base import Action
+
+
+class DeduplicatePermissionsMixin(Action):
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Deduplicate permissions
+        """
+        instance = super().update_instance(instance)
+        if instance.get("permissions") is None:
```
Comment (FinnStutzenstein) [inline openslides_backend/action/role/deduplicate_permissions_mixin.py:12]:
```suggestion
        if "permissions" in instance:
            instance["permissions"] = list(set(instance["permissions"]))
```

**Review by FinnStutzenstein — APPROVED** [review]
One little change suggested. @jsangmeister or @normanjaeckel may approve and merge, if everything is fine.

**Review by jsangmeister — APPROVED** [review]

File: openslides_backend/action/role/deduplicate_permissions_mixin.py (hunk 1 above)
Comment (r-peschke) [reply openslides_backend/action/role/deduplicate_permissions_mixin.py]: Done

**Pull request merged**